### DISCUSSION TITLE
Why does increasing dt result in an error?

### DISCUSSION THREAD

**Discussion by YXY101676** [idaholab/moose#28943] [Q&A Modules: Solid mechanics]
Dear MOOSE users,
   I am using Moose to calculate crystal plasticity, but the required deformation is large, so in order to speed up the calculation, I tried to dt, but the following error occurred:
![屏幕截图 2024-10-28 155258](https://github.com/user-attachments/assets/7ac70e9a-fad1-4fcd-ad6f-bd57659602d9)
Is there any way to accelerate the computation?
Best Regards,
YXY

**Comment by GiudGiud** [comment]
Hello

You could work on improving the preconditioning, or making the mesh / problem smaller. 
Large dt is not always safe in problems with deformation. 

Guillaume

**Reply by GiudGiud** [reply]
for explaining the error I ll need a backtrace. Use these instructions to generate that if it's not already displayed
https://mooseframework.inl.gov/application_development/debugging.html
